addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -1,5 +1,6 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from datetime import timedelta
```

Comment:
![image](https://github.com/user-attachments/assets/291193c6-2eb3-4bd6-a7b6-16f22c5a9dcd)
I would put it after the import logging